pkg/validation/cluster.go

```diff
@@ -607,6 +628,13 @@ func validateAzureCloudSpec(spec *kubermaticv1.AzureCloudSpec) error {
 			return err
 		}
 	}
+	if spec.NodePortsAllowedIPRanges != nil {
```

Comment:
nit: this block could be a function 
(but it is also nice to be able to see the whole code right here)

Reply:
good point